.github/workflows/tests.yml

```diff
@@ -72,6 +77,7 @@ jobs:
           npx jest --color --selectProjects "JavaScript behaviour tests"
 
       - name: Send screenshots to Percy
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
Do probably want a better way of doing this as this excluded all **JavaScript component tests** on Windows (not just Percy screenshots)

Reply:
Rebased with https://github.com/alphagov/govuk-frontend/pull/3009 so we can leave the Percy screenshot job as `ubuntu-latest` only